jackson-bom-pom.xml

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I suspect this got added accidentally. Since there is `jackson-bom` I don't think this is needed.